extensions/skyportal/static/js/components/ZTFAlert.jsx

```diff
@@ -0,0 +1,476 @@
+    {
+        id: 'id',
+        label: 'candid',
+        numeric: false,
+        disablePadding: false,
+        // minWidth: 170
+    },
+    {
+        id: 'jd',
+        numeric: true,
+        disablePadding: false,
+        label: 'JD',
+        // minWidth: 170,
+        align: 'left',
+        format: (value) => value.toFixed(5),
+    },
+    {
+        id: 'fid',
+        numeric: true,
+        disablePadding: false,
+        label: 'fid',
+        align: 'left',
+    },
+    {
+        id: 'mag',
+        numeric: true,
+        disablePadding: false,
+        label: 'mag',
+        // minWidth: 170,
+        align: 'left',
+        format: (value) => value.toFixed(3),
+    },
+    {
+        id: 'emag',
+        numeric: true,
+        disablePadding: false,
+        label: 'e_mag',
+        // minWidth: 170,
+        align: 'left',
+        format: (value) => value.toFixed(3),
+    },
+    {
+        id: 'rb',
+        numeric: true,
+        disablePadding: false,
+        label: 'rb score',
+        // minWidth: 170,
+        align: 'left',
+        format: (value) => value.toFixed(5),
+    },
+    {
+        id: 'drb',
+        numeric: true,
+        disablePadding: false,
+        label: 'drb score',
+        // minWidth: 170,
+        align: 'left',
+        format: (value) => value.toFixed(5),
+    },
+    {
+        id: 'isdiffpos',
+        numeric: false,
+        disablePadding: false,
+        label: 'isdiffpos',
+        // minWidth: 170,
+        align: 'left'
+    },
+    {
+        id: 'programid',
+        numeric: true,
+        disablePadding: false,
+        label: 'programid',
+        // minWidth: 170,
+        align: 'left'
+    },
+    {
+        id: 'alert_actions',
+        numeric: false,
+        disablePadding: false,
+        label: 'actions',
+        align: 'right',
+        // render: ({ row }) => <Link to={'lol/'}>lolol</Link>,
+        // render: ({ row }) => (<Link to={{ pathname: `/foo/${row.id}` }}>{row.label}</Link>)
+    },
+];
+
+
+function descendingComparator(a, b, orderBy) {
+    if (b[orderBy] < a[orderBy]) {
+        return -1;
+    }
+    if (b[orderBy] > a[orderBy]) {
+        return 1;
+    }
+    return 0;
+}
+
+function getComparator(order, orderBy) {
+    return order === 'desc'
+        ? (a, b) => descendingComparator(a, b, orderBy)
+        : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+function stableSort(array, comparator) {
+    const stabilizedThis = array.map((el, index) => [el, index]);
+    stabilizedThis.sort((a, b) => {
+        const order = comparator(a[0], b[0]);
+        if (order !== 0) return order;
+        return a[1] - b[1];
+    });
+    return stabilizedThis.map((el) => el[0]);
+}
+
+function EnhancedTableHead(props) {
+    const {classes, order, orderBy, onRequestSort} = props;
+    const createSortHandler = (property) => (event) => {
+        onRequestSort(event, property);
+    };
+
+    return (
+        <TableHead>
+            <TableRow>
```

Comment:
Are these indents of size 3?  I think we use 2.

Reply:
My pycharm was configured to use 4 for js code for some reason... Changed to 2